src/com/TTT/TurnUI.java

```diff
@@ -23,17 +23,34 @@ public void takeTurn(Board board, Game game) {
         String winningMarker = game.getPlayerMarker(game.currentPlayer());
 
         while(!validSelection(selectedSpace) || !game.selectionOnBoard(parseSelection(selectedSpace)) || board.isOccupied(parseSelection(selectedSpace))) {
```

Comment:
For readability it might be nice to wrap these multiple methods in another called `invalid(selectedSpace)`